package.json

```diff
@@ -31,102 +31,102 @@
   },
   "homepage": "https://github.com/TheDevs-Network/kutt#readme",
   "dependencies": {
-    "@bugsnag/js": "^7.20.2",
+    "@bugsnag/js": "^7.21.0",
     "@bugsnag/plugin-express": "^7.19.0",
     "app-root-path": "^3.1.0",
-    "axios": "^1.1.3",
+    "axios": "^1.6.1",
     "bcryptjs": "^2.4.3",
-    "bull": "^4.10.1",
+    "bull": "^4.11.5",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "cross-env": "^7.0.3",
     "d3-color": "^3.1.0",
-    "date-fns": "^2.29.3",
-    "dotenv": "^16.0.3",
-    "easy-peasy": "^5.1.0",
+    "date-fns": "^2.30.0",
+    "dotenv": "^16.3.1",
+    "easy-peasy": "^6.0.4",
     "email-validator": "^2.0.4",
-    "envalid": "^7.3.1",
+    "envalid": "^8.0.0",
     "express": "^4.18.2",
-    "express-async-handler": "1.1.4",
-    "express-validator": "^6.14.2",
-    "geoip-lite": "^1.4.6",
-    "helmet": "^6.0.0",
-    "ioredis": "^5.2.4",
-    "isbot": "^3.6.3",
-    "js-cookie": "^3.0.1",
-    "jsonwebtoken": "^8.5.1",
-    "jwt-decode": "^3.1.2",
-    "knex": "^2.3.0",
+    "express-async-handler": "1.2.0",
+    "express-validator": "^7.0.1",
+    "geoip-lite": "^1.4.8",
+    "helmet": "^7.1.0",
+    "ioredis": "^5.3.2",
+    "isbot": "^3.7.1",
+    "js-cookie": "^3.0.5",
+    "jsonwebtoken": "^9.0.2",
+    "jwt-decode": "^4.0.0",
+    "knex": "^3.0.1",
```

Comment:
This wont work for node 12 (current base image), v3 is compatible with node >=16. https://github.com/knex/knex/releases/tag/3.0.0

Reply:
Could we / should we bump it to node 21?

Reply:
probably bump to whatever is the latest LTS, i think that 18